### PR TITLE
Move cache client to open source

### PR DESCRIPTION
**Pull request by Ekrekr**

### CODE REVIEW

File: common/protos/proto_utils.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { util } from "protobufjs";
+
+export class ProtoUtils {
```
Comment (BenBirt) [inline common/protos/proto_utils.ts:3]: https://source.cloud.google.com/tada-analytics/github_dataform-co_dataform/+/master:common/protos/index.ts?q=f:protos%20lang:ts

File: common/strings/stringifier.ts [hunk 2]
```diff
@@ -282,3 +284,18 @@ export class StringifiedSet<T> implements Set<T> {
     })();
   }
 }
+
+export class ProtoStringifier<T> implements IStringifier<T> {
```
Comment (BenBirt) [inline common/strings/stringifier.ts:288]: this probably more cleanly belongs inside https://source.cloud.google.com/tada-analytics/github_dataform-co_dataform/+/master:common/protos/index.ts?q=f:protos%20lang:ts too.

Comment (Ekrekr) [reply common/strings/stringifier.ts:288]: I did that at first too, but it results in circular dependencies. Shouldn't bother for now I think

File: common/protos/index.ts [hunk 3]
```diff
@@ -50,3 +50,35 @@ function fromBase64(value: string): Uint8Array {
   util.base64.decode(value, buf, 0);
   return buf;
 }
+
+export class ProtoUtils {
```
Comment (BenBirt) [inline common/protos/index.ts:54]: sorry, i think i wasn't clear. this file already contains all of these utilities, i don't think we need `ProtoUtils` at all.

File: common/strings/stringifier.ts (hunk 2 above)
Comment (BenBirt) [reply common/strings/stringifier.ts:288]: wait, why? that would imply that `strings/stringifier.ts` somehow depends on `protos/index.ts`, it really shouldn't, how does it?

Comment (Ekrekr) [reply common/strings/stringifier.ts:288]: My bad, was making protos depend on itself, not strings

**Review by BenBirt — APPROVED** [review]

**Pull request merged**